src/components/views/dialogs/CreateRoomDialog.tsx

```diff
@@ -265,9 +268,13 @@ export default class CreateRoomDialog extends React.Component<IProps, IState> {
             </p>;
         } else if (this.state.joinRule === JoinRule.Invite) {
             publicPrivateLabel = <p>
-                { _t(
-                    "Only people invited will be able to find and join this room.",
-                ) }
+                { _t("Only people invited will be able to find and join this room.") }
+                &nbsp;
+                { _t("You can change this at any time from room settings.") }
```

Comment:
The `&nbsp;` would get merged into the other label, though I don't think the string needs changing here.

Reply:
Got it I'll change it

Reply:
oops, sorry - this comment shouldn't be here. Just ignore it.

Github forgot to delete it :(

Reply:
Np :)
I thought eslint detects these characters and is sensitive😅